nntrainer/layers/rnn.cpp

```diff
@@ -144,4 +144,9 @@ void RNNLayer::calcGradient() {
   // NYI
 }
 
+void RNNLayer::setActivation(ActivationType acti_type) {
+  Layer::setActivation(acti_type);
```

Comment:
please override `setProperty` and add this in `setProperty`.
`setActivation` will soon be removed.

Reply:
I will. Thanks.